openshift/e2e-tests-openshift.sh

```diff
@@ -196,7 +197,21 @@ EOF
 }
 
 function deploy_serverless_operator(){
-  oc apply -f openshift/serverless/operator-install.yaml
+  git clone https://github.com/openshift-knative/serverless-operator.git /tmp/serverless-operator
```

Comment:
Is it guaranteed that there is no `/tmp/serverless-operator` ? Better use `mktemp` for creating a temporary name and clean it up afterwards (store it in a var)

Reply:
IIUC, these operations are performed in a test pod which is a fresh one per PR test and goes away after the tests.